index.html

```diff
@@ -0,0 +1,102 @@
+<!DOCTYPE html>
+<html lang="KO">
```

Comment:
html은 대소문자를 구분하지 않기때문에 `KO`로 작성해주셔도 동작은 하는데요!
일반적으로 소문자로 작성하는 편이기 때문에 `ko`로 작성이 되면 더 좋을 것 같아요!

```suggestion
<html lang="ko">
```